include/clap/factory/preset-discovery.h

```diff
@@ -93,17 +99,6 @@ typedef uint64_t clap_timestamp_t;
 // Value for unknown timestamp.
 static const clap_timestamp_t CLAP_TIMESTAMP_UNKNOWN = 0;
```

Comment:
PR for renaming `clap_timestamp_t` and moving it to a separate header: #385 
(also CLAP_CONSTEPR was missing for `CLAP_TIMESTAMP_UNKNOWN`